Documentation/Home/ExtensionManuals.rst

```diff
@@ -0,0 +1,113 @@
+:template: extensions.html
+
+.. _extensions:
+
+=============================
+Developing & Using Extensions
+=============================
+
+.. ATTENTION:
+   Be careful with this special folder /typo3cms/extensions !!!
+
+.. Note the special template 'extensions.html' (see beginning of this file)
+
+.. First
+   You may add normal rst content here.
+
+Extension Development
+=====================
+
+.. container:: row m-0 p-0
+
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header
+
+            .. rubric:: Getting started with extension development
+
+         .. container:: card-body
+
+            *  :ref:`Introduction to extensions in TYPO3
+               <t3coreapi:extension-architecture-introduction>`
+
+            *  Extension :ref:`t3coreapi:extension-files-locations`
+
+            *  :ref:`t3coreapi:extension-naming` and :ref:`t3coreapi:cgl`
+
+
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header
+
+            .. rubric:: TYPO3 Explained - API Quicklinks
+
+         .. container:: card-body
+
+            *  :ref:`QueryBuilder <t3coreapi:database-query-builder>` based on Doctrine
+            *  :ref:`t3coreapi:DependencyInjection`
+            *  :ref:`t3coreapi:request-handling`
+
+Extension Manuals
+==================
+
+.. container:: row m-0 p-0
+
+   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
+
+      .. container:: card px-0 h-100
+
+         .. rst-class:: card-header
+
+            .. rubric:: System Extensions
+
+         .. container:: card-body
+
+            Here you can find documentation of system extensions and third party extensions,
+            if available.
```

Comment:
```suggestion
            Here you can find documentation for system extensions, as well as third party extensions,
            if available.
```

Reply:
Oh woops - you mean we're still missing docs for some system extensions? Ignore my suggestion then.

Reply:
Yes, sadly we do :/